static/app/views/explore/toolbar/index.tsx

```diff
@@ -44,9 +44,9 @@ export function ExploreToolbar({extras, width}: ExploreToolbarProps) {
       {extras?.includes('dataset toggle') && (
         <ToolbarDataset dataset={dataset} setDataset={setDataset} />
       )}
-      <ToolbarMode mode={mode} setMode={setMode} />
+      {!extras?.includes('tabs') && <ToolbarMode mode={mode} setMode={setMode} />}
       <ToolbarVisualize equationSupport={extras?.includes('equations')} />
-      {mode === Mode.AGGREGATE && <ToolbarGroupBy />}
+      {(extras?.includes('tabs') || mode === Mode.AGGREGATE) && <ToolbarGroupBy />}
```

Comment:
I noticed this field does nothing in this PR until you switch to aggregate mode in the tabs. I assume this is because you mentioned wanting to put in a mini aggregate table below the chart, but wanted to call it out in case that wasn't the intention of this change.

Reply:
I was asked to make sure this change was behind a feature flag so we can test it prior to releasing it. This line lines if the tabs ui is not enabled, we fall back to the current behaviour of only showing the group by toolbar in aggregate mode.